srcs/web/WebhookManager.py

```diff
@@ -49,6 +49,29 @@ def workflow_job(self, payload: WebHook):
             or "self-hosted" not in payload.workflow_job.labels
             or payload.workflow_job.runner_id is None
         ):
+            payload.workflow_job.labels.remove("self-hosted")
+            try:
+                r_m = next(
+                    iter(
+                        runner_m.get_runner_manager_on_demand(
+                            lambda elem: elem.vm_type.tags
+                            == payload.workflow_job.labels
+                        )
+                    ),
+                    None,
+                )
+            except Exception as e:
+                logger.info(e)
```

Comment:
Should it be an error here or nah?
```suggestion
                logger.error(e)
```